src/app/features/course/search/search.component.spec.ts

```diff
@@ -22,4 +24,25 @@ describe('SearchComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('search() should be called', () => {
+    spyOn(component, 'search');
+
+    const button = fixture.debugElement.nativeElement.querySelector('.search__button');
+    button.click();
+    expect(component.search).toHaveBeenCalled();
+  });
+
+  it('search() should call filterData.emit(term)', () => {
```

Comment:
Usualy 'it' describe starts from word 'should'. For example 'should call emit in `search` method'. But better 'should emit data (where)'

Reply:
Suggestion implemented, thanks